en/docs/api-analytics/on-prem/elk-installation-guide.md

```diff
@@ -191,6 +191,31 @@ Open the `wso2am-4.x.x/repository/conf` directory. To enable logging for a repor
     | "errorMessage"               | string    | The error message associated with the fault.                         |
     | "errorType"                  | string    | The type of error (e.g., THROTTLED).                                 |
 
+    
+#### Step 1.3 - Enabling Operation Policies in Analytics Event
+
+Operation and API Policies of the APIs/API Products are loaded to the Analytics Event by enabling the following property. This can be either provide as an argument or configured in the start up scripts on the server startup.
```

Comment:
```suggestion
Operation and API Policies of the APIs/API Products are loaded to the Analytics Event by enabling the following property. This can either be provided as an argument at the server startup or configured in the startup scripts.
```